docs/user/sbt.rst

```diff
@@ -215,6 +215,26 @@ Use the following approach in `sbt` to set the target triple:
 
     nativeConfig ~= { _.withTargetTriple("x86_64-apple-macosx10.14.0") }
 
+you may create a few dedicated projects with different target triple.
+For example if I have two `sandbox` project like:
```

Comment:
Since lipo is macOS specific we might include this information here
```suggestion
For example, if I have multiple project definitions for different macOS architectures, eg:
```